h/templates/includes/paginator.html.jinja2

```diff
@@ -1,33 +1,47 @@
 <nav>
-  <ul class="pagination">
+  <ul class="pager">
     {% if page.prev is none %}
-      <li class="disabled">
-        <span aria-hidden="true">&laquo;</span>
+      <li class="item-disabled pager__item item-begin">
+        <span aria-hidden="true">&#x3c;</span>
```

Comment:
As far as I am aware, all of our source files use UTF-8 encoding, so I think it should be possible to just use the `<` character literally, rather than a hex escape.